packages/node/src/dev-server.mts

```diff
@@ -6,13 +6,14 @@ if (!entrypoint) {
 }
 
 import { join } from 'path';
-import type { Headers } from 'node-fetch';
+import type { Headers } from 'undici';
 import type { VercelProxyResponse } from './types.js';
 import { Config } from '@vercel/build-utils';
 import { createEdgeEventHandler } from './edge-functions/edge-handler.mjs';
 import { createServer, IncomingMessage, ServerResponse } from 'http';
 import { createServerlessEventHandler } from './serverless-functions/serverless-handler.mjs';
 import { isEdgeRuntime, logError, validateConfiguredRuntime } from './utils.js';
+import { toToReadable } from '@edge-runtime/node-utils';
```

Comment:
Why is this called `toToReadable`?

Shouldn't it be called `toReadable` instead?

Reply:
the good new is... it can be dropped too!

https://github.com/vercel/edge-runtime/blob/bbce361d6f771415996ca15bb46790f34b5b0d74/packages/node-utils/src/edge-to-node/stream.ts#L11

Reply:
`stream.Readable.fromWeb` was added in v17 so it's missing in v16 <picture data-single-emoji=":oh_no:" title=":oh_no:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/oh_no/02f810a2c304447f.png" alt=":oh_no:" align="absmiddle"></picture>